app/api.py

```diff
@@ -651,3 +656,52 @@ def list_network_test_results_api(self, from_timestamp: Union[str, datetime.date
             except Exception as e:
                 logging.error(f"Error in reachability test results: {e}")
                 return
+    
+    def list_client_type_distribution_api(self, from_timestamp: Union[str, datetime.datetime],
+                                      to_timestamp: Union[str, datetime.datetime]):
+        """
+        List Client Type distribution API
+
+        :param from_timestamp: Union[str, datetime.datetime]
+        :param to_timestamp: Union[str, datetime.datetime]
+        :param deviceType: str
```

Comment:
`deviceType` parameter has not been passed but it is mentioned in the docstring here.